content/en/security/application_security/troubleshooting.md

```diff
@@ -564,28 +564,41 @@ See the Application Security [set up documentation][11] to validate you you are
 
 Ensure the `DD_INSTRUMENTATION_TELEMETRY_ENABLED` environment variable (`DD_TRACE_TELEMETRY_ENABLED` for NodeJS) is set to `true`, or the corresponding system property for your language is enabled. For example in Java: `-Ddd.instrumentation.telemetry.enabled=true`
 
-## Disabling Threat Detection and Protection
+## Disabling threat detection and protection
 
 To disable ASM, remove the `DD_APPSEC_ENABLED=true` environment variable from your application configuration, and restart your service.
 
 If no `DD_APPSEC_ENABLED=true` environment variable is set for your service:
 * If it's a PHP service: explicitly set the environment variable to `DD_APPSEC_ENABLED=false`, and restart your service.
-* If ASM was activated in one click: go to [ASM service configuration][15], select **Services with Threat Management Capabilities**, click on your service, and then click **Deactivate**.
-* To disable ASM on your services in bulk, click the check box in the list header and then under **Bulk Actions** select **Deactivate Threat Management detection on (number of) services**.
-
-## Disabling Software Composition Analysis (SCA)
+* If ASM was activated in one click, do the following: 
```

Comment:
It might be interested to let the customer know how to check which case they fall in. 

The deactivation button will only be there if it was activated in 1-click, If the UI redirects to the docs, ASM wasn't activated in 1-click. 
<img width="1419" alt="image" src="https://github.com/DataDog/documentation/assets/11967750/bb9c607b-51fc-4a5d-98c0-b4d060c799dd">